src/extensions/userSpacing.js

```diff
@@ -55,8 +55,8 @@ const userTextSpacingWithLineHeight = lineHeight => props => {
    * smaller or larger, so we further scale the value based on its difference
    * from 1.5. Experiments suggest this produces reasonably accurate results.
    */
-  const topOffset = `${lineHeight / -6.0 * (lineHeight / 1.5)}em`;
-  const bottomOffset = `${lineHeight / -3.0 * (lineHeight / 1.5)}em`;
+  const topOffset = `${lineHeight / -5.25 * (lineHeight / 1.5)}em`;
```

Comment:
Is `-5.25 ` and `-3.5` (below) proportional to something? I'm nervous these values are going to be proportionate to  a particular width and font-size, meaning it will work only in certain situations (which might also mean that the values above worked for a particular page)